lib/combineMeshes.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+module.exports = combineMeshes;
+
+//Combines meshes in a node.
+function combineMeshes(gltf) {

```

Comment:
This needs to run before the `combinePrimitives` stage, right?  This creates just one mesh with a big array with all the primitives, then it is expected that `combinePrimitives` will combine primitives.

I think it is worth adding a comment here.